src/model.js

```diff
@@ -172,14 +171,14 @@ export default class Model {
             }))
         );
 
-        // Добавляем измененные страницы, предварительно внедряя изменения которые
-        // пришли и новой модели
+        // Добавляем измененные страницы, предварительно внедряя изменения, которые
+        // пришли, и новой модели
         // merge modifications
         // add modified pages
         this.setPages(
             this.getPages().concat(modifiedPages.map(url => {
-                this.pushChangeModify({type: 'page', url});
-                return deepExtend(oldModel[url], newModel[url]);
+                this.pushChangeToModifiedGroup({type: 'page', url});
```

Comment:
'object short notation' is available in ES6 (use esnext option) or Mozilla JS extensions (use moz).